bidict/_loose.py

```diff
@@ -0,0 +1,9 @@
+from ._bidict import bidict
+
+class loosebidict(bidict):
+    """
+    A mutable bidict which always uses forcing put operations
+    so that it never raises :class:`ValueExistsException`.
+    """
+    def _put(self, key, val):
```

Comment:
Tiny performance optimisation, but

    _put = bidict.forceput

would remove function call overhead without increasing code duplication. I imagine _put() will get called a lot in loops, so this adds up.